jailbreak.h

```diff
@@ -4,17 +4,23 @@
 
 struct jbc_cred
 {
-    uid_t uid;
-    uid_t ruid;
-    uid_t svuid;
-    gid_t rgid;
-    gid_t svgid;
-    uintptr_t prison;
-    uintptr_t cdir;
-    uintptr_t rdir;
-    uintptr_t jdir;
+	uid_t uid;
+	uid_t ruid;
+	uid_t svuid;
+	gid_t rgid;
+	gid_t svgid;
+	uintptr_t prison;
+	uintptr_t cdir;
+	uintptr_t rdir;
+	uintptr_t jdir;
+	uint64_t sonyCred;
```

Comment:
Suggestion: change order of fields to the same as in PS4 kernel, so that the values can be copied in a single ppcopy{in,out}. This should not really matter, but feels less optimal.